pkg/controller/controller.go

```diff
@@ -70,10 +69,8 @@ type Controller struct {
 }
 
 type Config struct {
-	Namespace     string
-	MasterHost    string
-	KubeCli       *unversioned.Client
-	PVProvisioner string
+	MasterHost string
+	cluster.Config
```

Comment:
Cool. Let's duplicate them.

In my experience, having common struct when it's unstable wouldn't be a good idea.